stable/redmine/values.schema.json

```diff
@@ -0,0 +1,196 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "redmineUsername": {
+      "type": "string",
+      "title": "Username",
+      "form": true
+    },
+    "redminePassword": {
+      "type": "string",
+      "title": "Password",
+      "form": true,
+      "description": "Defaults to a random 10-character alphanumeric string if not set"
+    },
+    "redmineEmail": {
+      "type": "string",
+      "title": "Admin email",
+      "form": true
+    },
+    "persistence": {
+      "type": "object",
+      "title": "Persistence for Redmine replicas",
+      "form": true,
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "form": true,
+          "title": "Enable persistence",
+          "description": "Enable persistence using Persistent Volume Claims"
+        },
+        "size": {
+          "type": "string",
+          "title": "Persistent Volume Size",
+          "form": true,
+          "render": "slider",
+          "sliderMin": 1,
+          "sliderMax": 100,
+          "sliderUnit": "Gi",
+          "hidden": {
+            "condition": false,
+            "value": "persistence.enabled"
+          }
+        }
+      }
+    },
+    "ingress": {
+      "type": "object",
+      "form": true,
+      "title": "Ingress Details",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "form": true,
+          "title": "Use a custom hostname",
+          "description": "Enable the ingress resource that allows you to access the Redmine installation."
+        },
+        "hostname": {
+          "type": "string",
+          "form": true,
+          "title": "Hostname",
+          "hidden": {
+            "condition": false,
+            "value": "ingress.enabled"
+          }
+        }
+      }
+    },
+    "databaseType": {
```

Comment:
maybe this is a problem of the chart but why is there `databaseType.mariadb` and `mariadb.enabled`? Isn't that for the same thing?

Also, in case you want to keep `databaseType`, you may want to make it a string with two allowed values: `mariadb|postgresql|external`. You can force that with the verification :)

Reply:
> maybe this is a problem of the chart but why is there databaseType.mariadb and mariadb.enabled? Isn't that for the same thing?

We use `databaseType` to decide whether to use MariaDB or PostgreSQL as database. On the other hand, we use `mariadb.enabled` to decide whether we should use the MariaDB chart to create a database in the cluster or not.

> Also, in case you want to keep databaseType, you may want to make it a string with two allowed values: mariadb|postgresql|external. You can force that with the verification :)

That's a good suggestion. I'll apply it

Reply:
> > maybe this is a problem of the chart but why is there databaseType.mariadb and mariadb.enabled? Isn't that for the same thing?
> 
> We use `databaseType` to decide whether to use MariaDB or PostgreSQL as database. On the other hand, we use `mariadb.enabled` to decide whether we should use the MariaDB chart to create a database in the cluster or not.
> 

What would happen if you set `databaseType.mariadb` and `mariadb.enabled=false` at the same time? Isn't that an inconsistent status? What I mean, is there a valid use case in which those two parameters may not match?